scripts/prove_rpc.sh

```diff
@@ -47,6 +47,7 @@ NODE_RPC_TYPE=$4
 IGNORE_PREVIOUS_PROOFS=$5
 BACKOFF=${6:-0}
 RETRIES=${7:-0}
+TEST_ONLY=$8
```

Comment:
Argument at ix 7 is optional, but argument at 8 is required. Is this what you intend?

Reply:
I haven't changed anything in this file, I've just made it as an explicit variable for clarity below (we were previously having `if [[ $8 == "test_only" ]]`)